_source/_posts/2021-10-16-full-stack-java.md

```diff
@@ -0,0 +1,693 @@
+---
+layout: blog_post
+title: "Full Stack Java with React, Spring Boot, and JHipster"
+author: matt-raible
+by: advocate
+communities: [java,javascript]
+description: "This tutorial shows you how to be a full stack Java developer with React and Spring Boot."
```

Comment:
there is a similar issue with this description as in the title. I think this tutorial shows rather how to create a fullstack app with Jhipster/React/SpringBoot rather than how to be a full stack java dev, because I think it requires a lot more than this article ;)